src/jobQueue.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  run,
+  quickAddJob,
+} from 'graphile-worker';
+import { processBulkUpload } from './tasks';
+import type { ProcessBulkUploadJobPayload } from './types';
+
+enum JobType {
+  PROCESS_BULK_UPLOAD = 'processBulkUpload',
+}
+
+export const startJobQueue = async () => {
+  const runner = await run({
```

Comment:
I think we want to pass a logger as part of this invocation. From the [documentation on logging](https://worker.graphile.org/docs/library/logger), it looks like it's just a function that gets passed in as part of the `run` argument object.